lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -324,7 +324,7 @@ def create_tar(cv, cvv, repositories, json)
 
         Dir.mkdir("#{export_dir}/#{export_prefix}")
 
-        if repositories
+        if repositories.any?
```

Comment:
`s/repositories.any?/repositories&.any?/` will fix your tests

Reply:
@akofink Thanks that worked